slackbot.sh

```diff
@@ -10,6 +10,69 @@
 #  * SLACK_URL_RELEASE The slack webhook url for the appropriate channel to post to
 set -e
 
+function usage()
+{
+    echo "News UK github release slack notification bot"
+    echo ""
+    echo "./slackbot.sh"
+    echo "\t-h --help"
+    echo "\tPrints this help"
+    echo ""
+    echo "\t--git-repo-url"
+    echo "\tThe git repository URL (e.g. git@github.com:newsuk/project.git)"
+    echo ""
+    echo "\t--git-hash"
+    echo "\tThe commit hash of the build, must be tagged!"
+    echo ""
+    echo "\t--project-name"
+    echo "\tThe project name"
+    echo ""
+    echo "\t--released-by"
+    echo "\tThe user who released"
+    echo ""
+    echo "\t--slack-url-release"
+    echo "\tThe slack webhook url for the appropriate channel to post to"
+    echo ""
+    echo "\t--release"
+    echo "\tPatches the github release status from prelrelease to release"
+    echo ""
+}
+
+while [ "$1" != "" ]; do
```

Comment:
I did it to make the release status patch a commandline argument and disabled by default (so the bot doesn't change the release status). Since I was adding commandline argument I figured I could easily add all the other environment vars a commandline arguments too.

I agree changing the release status should be a feature of the release tool but at the moment it requires repo checkout

Reply:
OK I see, it just complicated the picture a little bit for me was all.